LSE-509.tex

```diff
@@ -100,55 +101,59 @@
 \section{Project Organization in System AI\&T and Commissioning}
 \label{sec:project_organization}
 
-During system assembly, integration and test (AI\&T) and commissioning, the overall Project organizational structure remains unchanged.
+During system assembly, integration and test (AI\&T) and commissioning, the overall Project WBS organizational structure remains unchanged.
 The Project Director, Deputy Director and Project Manager retain overall responsibility and authority as described in the Project Execution Plan (PEP) (\citeds{LPM-54}).
-The System Assembly, Integration, Test and Commissioning (SIT-Com) team has been formed and is led by the Systems Scientist.
-The high-level description of this position is to plan and execute the observatory commissioning effort, ensuring an on-time and on-budget delivery.
-The Systems Scientist has the authority to direct resources as required to meet the functional objectives described in this plan and are ultimately responsible for all aspects of SIT-Com.
+However, as the Project enters into its final stages some functional change is required, therefore the System Assembly, Integration, Test and Commissioning (SIT-Com) team has been formed and is led by the Systems Scientist.
+The high-level description of this team is to plan, execute and lead the observatory commissioning effort, ensuring an on-time and on-budget delivery meeting the (\href{https://sitcomtn-005.lsst.io}{Construction Completeness Criteria }).
+The Systems Scientist has the authority to direct resources as required to meet the functional objectives described in this plan and is ultimately responsible for all aspects of SIT-Com.
 
-The SIT-Com framework, which was created relatively recently in comparison to the other subsystems in the project, has grown out of and now encompasses the System's Engineering (SE) group.
-Consequently, SIT-Com is now responsible for the verification, validation, and science verification activities, however they remain under the auspices of the SE group.
-The SE team management plan (\citeds{LSE-17}) details the internal workings of that group and details the responsibilities and adopted methodologies, including verification procedures and tooling.
+The SIT-Com team framework has grown out of and now encompasses both Project System's Engineering (PSE) and Commissioning.
+With the integration of Commissioning and Systems Engineering groups, SIT-Com is now responsible for the verification, validation, commissioning and science validation activities.
+The PSE team management plan (\citeds{LSE-17}) details the internal workings of that group and details the responsibilities and adopted methodologies, including verification procedures and tooling.
 More DOE specific content is found in the Commissioning Execution Plan (\citeds{LSE-390}).
+The purpose of this document is to describe the details, roles and responsibilities of the overall SIT-Com team.
```

Comment:
what is "this document" ? I think you mean LSE-509 but as written it sounds like LSE-390.
Maybe just put LSE-509 in brackets after "this"?